app/components/List/tests/index.test.js

```diff
@@ -18,9 +25,10 @@ describe('<List />', () => {
       'Hello',
       'World',
     ];
-    const renderedComponent = render(
+    const renderedComponent = mount(
       <List items={items} component={ListItem} />
     );
-    expect(renderedComponent.find(items)).toExist();
+    expect(renderedComponent.find(ListItem)).toExist();
+    expect(renderedComponent.find(ListItem).length).toEqual(items.length);
```

Comment:
I'd recommend using `.toEqual(2)` here, it's always better to be explicit when checking equality in tests.
In this case the 'items' array is explicitly set and there are no side effects. It's also quite clear that no values are added or removed from the items array but that's not always the case...
Imagine if for some crazy reason the length of items changed during the mounting of the component, leaving items.length as 10, we would be checking that the expected value is equal to 10 without knowing.
I hope that makes sense, it's quite difficult to explain!